user.rb

```diff
@@ -17,13 +18,14 @@ def self.all
     users_json = File.exist?("users.json") ? File.read("users.json") : "[]"
     parsed_users = JSON.parse(users_json, symbolize_names: true)
     parsed_users.map do |user|
-      User.new(first_name: user[:first_name], last_name: user[:last_name])
+      User.new(first_name: user[:first_name], last_name: user[:last_name], email: user[:email])
```

Comment:
This is just a nit that can be ignored: At some point it may make sense to move these attribute setters to new lines like you have in `User.create` above:

https://github.com/steveryan/file_based_database/blob/b65c99da69d63f303b879db655d0fe9d99bd3438/user.rb#L7-L11

Reply:
yeah that makes sense